src/views/Rewards/WeekDropdown.js

```diff
@@ -28,7 +33,17 @@ export default function WeekDropdown(props) {
                   .sort((a, b) => b.week - a.week)
                   .map((rewardWeek) => (
                     <Menu.Item>
-                      <div className="menu-item large" onClick={() => setSelectedWeek(parseFloat(rewardWeek.week) + 1)}>
+                      <div
+                        className="menu-item large"
+                        onClick={() => {
+                          const selectedWeek = parseFloat(rewardWeek.week) + 1;
+                          setSelectedWeek(selectedWeek);
+                          trackAction("Button clicked", {
+                            buttonName: "Select rewards week",
+                            weekNo: selectedWeek,
+                          });
+                        }}
+                      >
```

Comment:
is this offset accounted for when selecting `allWeeks[selectedWeek]`

Reply:
Will be now, ty for the pickup

Reply:
yes, its re-adjusted in L81 in `Rewards.js`:

![image](https://user-images.githubusercontent.com/19278287/183001168-68df64a5-a8f4-4467-adef-9eac200ddf56.png)

Reply:
if you know a more efficient way of incrementing the week in the UI but retaining the week index for the array, let me know